lib/cpus/aarch64/denver.S

```diff
@@ -267,27 +267,27 @@ func denver_cpu_reg_dump
 	ret
 endfunc denver_cpu_reg_dump
 
-declare_cpu_ops denver, DENVER_MIDR_PN0, \
+declare_cpu_ops_workaround_cve_2017_5715 denver, DENVER_MIDR_PN0, \
 	denver_reset_func, \
 	denver_core_pwr_dwn, \
```

Comment:
you need to set a check function before core_pwr_dwn that returns #ERRATA_APPLIES when the cpu is vulnerable, #ERRATA_NOT_APPLIES if it is not vulnerable and #ERRATA_MISSING if the cpu is vulnerable but mitigation is not enabled.

ditto for all macros below.

Reply:
the reason it builds is because the macro has variadic arguments